### DISCUSSION TITLE
Ergo pool - high fixed difficulty issue

### DISCUSSION THREAD

**Discussion by dbaldree** [coinfoundry/miningcore#1032] [Q&A] [answered]
Hi team,

Pool appears to be working however when you try to target Nicehash or set a fixed difficulty above '40000' the RPC calls bail out.
Looks like the bignum function or something around sharedifficulty is limited.

20211208 15:35:32 Authorized successfully.
20211208 15:35:32 WARN: Can't handle stratum method in payload {"jsonrpc":"2.0","method":"mining.notify","params":["00000b55",637079,"very-long-hash-here","","",2,"0","",true],"id":null}
, Base 8 or 16 printing of negative numbers is not supported.

Setting d=40000 or lower in the password in the miner.cmd seems to look fine but anything higher and this breaks.

**Comment by oliverw** [comment, marked as answer]
Miningcore Ergo Difficulty 1.0 = 4 Billion

**Reply by dbaldree** [reply]
Oh boy -- sorry mate feel like a lemon. All is good thank you.

**Reply by oliverw** [reply]
> Oh boy -- sorry mate feel like a lemon. All is good thank you.

Why? Its not documented, how could you anticipate that?
